tools/link.py

```diff
@@ -656,7 +656,11 @@ def phase_linker_setup(options, state, newargs):
   if options.emrun:
     options.pre_js.append(utils.path_from_root('src/emrun_prejs.js'))
     options.post_js.append(utils.path_from_root('src/emrun_postjs.js'))
+    if settings.MINIMAL_RUNTIME:
+      exit_with_error('--emrun is not compatible with MINIMAL_RUNTIME')
     # emrun mode waits on program exit
+    if user_settings.get('EXIT_RUNTIME') == '0' or user_settings.get('NO_EXIT_RUNTIME') == '1':
```

Comment:
We normalize this, so `NO_EXIT_RUNTIME` will never be a key.

Reply:
Hmm, you are right, not sure what I was thinking there.